tests/python/unittest/test_operator.py

```diff
@@ -191,13 +191,12 @@ def test_rnnrelu_sym():
     stack.add(mx.rnn.RNNCell(H, activation='relu', prefix='l1_'))
     stack.add(mx.rnn.RNNCell(H, activation='relu', prefix='l2_'))
 
-    check_rnn_consistency(fused, stack, T, N, I, H, 'write')
-    check_rnn_consistency(fused, stack, T, N, I, H, 'add')
-    check_rnn_consistency(fused, stack, T, N, I, H, 'null')
-
+    check_rnn_consistency(fused, stack, T, N, I, H, 'write', rtol=1e-2, atol=1e-2)
+    check_rnn_consistency(fused, stack, T, N, I, H, 'add', rtol=1e-2, atol=1e-2)
+    check_rnn_consistency(fused, stack, T, N, I, H, 'null', rtol=1e-2, atol=1e-2)
 
```

Comment:
Do we need to change the tolerance for test_rnnrelu_sym too? test_rnnrelu_sym passes for 10000 runs with the existing code (rtol=1e-2, atol=1e-4). Only test_rnnrelu_bidirectional fails with this tolerance.

Reply:
just change back tolerance for test_rnnrelu_sym . thanks!